src/setup/rgw.py

```diff
@@ -359,7 +359,7 @@ def _update_rgw_config_with_endpoints(conf: MappedConf, endpoints: dict, instanc
         # Update this with same config that is define for 1st instance.
         if instance == 1:
             radosgw_admin_log_file = os.path.join(
-                log_path, COMPONENT_NAME, Rgw._machine_id, 'radosgw-admin.log')
+                log_path, 'radosgw-admin.log')
```

Comment:
Earlier below was the file path getting updated in conf file
> log file = /etc/cortx/log/rgw/e884506cadb150157e923ad8e4873ac4/rgw/e884506cadb150157e923ad8e4873ac4/radosgw-admin.log

updated the file path:

> log file = /etc/cortx/log/rgw/e884506cadb150157e923ad8e4873ac4/radosgw-admin.log